README.md

```diff
@@ -3,3 +3,40 @@
 [![codecov](https://codecov.io/gh/kalmarek/SymbolicWedderburn.jl/branch/master/graph/badge.svg)](https://codecov.io/gh/kalmarek/SymbolicWedderburn.jl)
 
 Amazing package providing symbolic Wedderburn decompositions for group *-algebras.
+Group *-algebras are closely related to
+* linear actions of **finite groups** on **finite dimensional** `k`-vector space `V` (i.e. a `k(G)`-module structure on `V`), which can be expressed in the language of
+* representations `ρ : G → GL(V)` of a (finite) group `G` on a (finite dimensional) `k`-vector space `V`.
+
+These objects naturally arise from (non)commutative polynomial optimization with symmetry and the aim of the package is to faciliate such uses.
+
+By Masche's theorem, `V ≅ m₁V₁ ⊕ ⋯ ⊕ mᵣVᵣ ≅ W₁ ⊕ ⋯ ⊕ Wᵣ` can be decomposed uniquely into isotypic/semisimple components `Wᵢ` and nonuniquely into irreducible/simple components `Vᵢ` associated with a group G. By (symbolic) computation in `k(G)`, `SymbolicWedderburn.jl` is capable of producing exact
+* isomorphism `V ≅ W₁ ⊕ ⋯ ⊕ Wᵣ`, or
+* projection `V → V₁ ⊕ ⋯ ⊕ Vᵣ`.
+
+The isomorphism gives a decomposition of `End_G(V)` in the sense of Wedderburn-Artin theorem. The projection is enough to reconstruct any matrix `P ∈ End_G(V)'` in the commutant of the endomorphism algebra.
+
+In case of problems of positive semi-definite optimization which enjoy group symmetry this decomposition can be used to reduce an **invariant** psd constraint
```

Comment:
```suggestion
In case of problems of positive semidefinite optimization which enjoy group symmetry this decomposition can be used to reduce an **invariant** psd constraint
```